webclient/app/src/app/commons/appTemplateCard/AppTemplateCard.jsx

```diff
@@ -73,29 +72,19 @@ function AppTemplateCard(props) {
             .then(() => {
                 handleRefresh();
             }).catch(err => {
+                setOpenErrorDialog(true);
+                setOpenDeleteDialog(false);
```

Comment:
Why is this error in a different Dialog? This could be confusing for the user and complicated if you want to re submit the request

Reply:
Der delete-dialog ist einfach nur ein confirmation dialog. Und das andere ist ein error Dialog. Ticket ab#43 erstellt für Snackbar.